x/metadata/keeper/query_server.go

```diff
@@ -649,6 +649,35 @@ func (k Keeper) ScopeSpecification(c context.Context, req *types.ScopeSpecificat
 		retval.ScopeSpecification = types.WrapScopeSpecNotFound(specAddr)
 	}
 
+	if found && req.IncludeContractSpecs {
+		for _, id := range spec.ContractSpecIds {
+			cs, ok := k.GetContractSpecification(ctx, id)
+			if !ok {
+				return &retval, fmt.Errorf("error iterating scope spec [%s] contract specs: %w", spec.SpecificationId, err)
```

Comment:
```suggestion
				return &retval, fmt.Errorf("error getting contract spec [%s]: %w", spec.SpecificationId, err)
```

Reply:
ah, this actually needs to be `id` instead of `spec.SecificcationId`